xwiki-enterprise-ui/xwiki-enterprise-ui-mainwikipreferences/src/main/resources/XWiki/XWikiPreferences.xml

```diff
@@ -2781,7 +2781,7 @@ Your account on ${wikiname} has been successfully activated. You can now login a
       <registration_registered>---</registration_registered>
     </property>
     <property>
-      <rightPanels>Panels.QuickLinks,Panels.MyRecentModifications</rightPanels>
+      <rightPanels>Panels.MyRecentModifications</rightPanels>
```

Comment:
This is going to be an issue because it means we won't have any panel in the panel bar for guest users (or if the currently logged in user has no modified pages) which will look odd.